requirements/partial/requirements_production.txt

```diff
@@ -7,8 +7,7 @@ fastjsonschema==2.19.1
 Flask==3.0.3
 gunicorn==22.0.0
 lxml==5.1.0
-psycopg==3.1.18
-psycopg2==2.9.9
+psycopg[pool]==3.1.18
```

Comment:
Why?

Reply:
psycopg[c] gives us the faster version of directly used libraries and a c-written parts for velocity
The separate psycopg-pool is recommended, because it is independent from psycopg in its versioning than the bundled version. See https://www.psycopg.org/psycopg3/docs/basic/install.html#binary-installation and following chapters.

Reply:
ok